packages/python-packages/doc-warden/warden/cmd_entry.py

```diff
@@ -42,16 +43,25 @@ def index(config):
 
 # verify the content of the readmes only
 def verify_content(config):
-    content_results, ignored_content_results = verify_readme_content(config)
-    output_content_results(content_results, config)
+    if 'readme' in config.target_files[0]:
+        content_results, ignored_content_results = verify_readme_content(config)
+        output_content_results(content_results, config)
 
-    if len(content_results) > 0:
-        conclusion_message(config)
-        exit(1)
+        if len(content_results) > 0:
+            conclusion_message(config)
+            exit(1)
 
-# verify the presence of the readmes only
+    if 'changelog' or 'history' in config.target_files[0]:
```

Comment:
@scbedd do you know of any reason that python uses history instead of changelog? Are they opposed to moving to changelog for consistency with everyone else?

Reply:
Do you mean changelog.md or just changelog.rst. I looked into renaming all histroy.rst to changelog.rst will be trivia. But changing to changelog.md will required changing the format of the files which doesn't seem like a very good idea.

Reply:
Tracked with this issue https://github.com/Azure/azure-sdk-for-python/issues/9096